targets/chrome-common

```diff
@@ -31,26 +31,52 @@ REQUIRES='x11'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Only download if it's not already installed, since it auto-updates
 pkg="google-chrome-$CHANNEL"
-if [ -n "`list_uninstalled '' "$pkg"`" ]; then
-    # Chrome needs hicolor-icon-theme to configure/install
-    install hicolor-icon-theme
-
-    url="https://dl.google.com/linux/direct/${pkg}_current_$ARCH.$PKGEXT"
-    tmp="`mktemp crouton.XXXXXX --tmpdir=/tmp`"
-    addtrap "rm -f '$tmp'"
-
-    wget "$url" -O "$tmp"
-
-    # Remove other providers of google-chrome, then install the package
-    for c in stable beta dev; do
-        c="google-chrome-$c"
-        if [ -z "`list_uninstalled '' "$c"`" ]; then
-            remove "$c"
-        fi
-    done
-    install_pkg "$tmp"
+
+# Chrome needs hicolor-icon-theme to configure/install
+install hicolor-icon-theme
+
+# Use the package manager for gentoo, it maintains chrome as well
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Rename google-chrome-stable to google chrome to match the ebuild
+    if [ "$CHANNEL" = 'stable' ]; then
+        GENTOO_PKG="www-client/google-chrome"
+    else
+        GENTOO_PKG="www-client/$pkg"
+    fi
+
+    # Keyword Chrome
+    keyword_package "$GENTOO_PKG" "$pkg"
+
+    # Accept the license
+    accept_license "$GENTOO_PKG" google-chrome "$pkg"
```

Comment:
On Gentoo, you accept licenses by adding the license that you've accepted into the /etc/portage/package.licenses file, as it's done by the accept_license function. Alternatively, you'd use the --autounmask-write capability of emerge to write out these changes and then update the file using etc-update or a similar configuration merging tool.

I suppose I could ask the user if they accept the license as part of the install sequence? Is it okay to have this interaction in the install process?

To be fair though, shouldn't that really happen on the other distros too?